src/Sign.Cli/AzureKeyVaultCommand.cs

```diff
@@ -14,6 +14,7 @@ namespace Sign.Cli
     internal sealed class AzureKeyVaultCommand : Command
     {
         internal Option<Uri> UrlOption { get; } = new(["--azure-key-vault-url", "-kvu"], AzureKeyVaultResources.UrlOptionDescription);
+        internal Option<string> CertificateVersionOption { get; } = new(["--azure-key-vault-certificate-version", "-kvuv"], AzureKeyVaultResources.CertificateVersionOptionDescription);
```

Comment:
This should probably be `-kvcv` instead of `-kvuv`? And there also also tests for the options in the `ParserTests` of `AzureKeyVaultCommandTests`. You could probably add an extra tests there.

Reply:
Changed the abbreviation value, added some test cases to AzureKeyVaultCommandTests to ensure addition or subtraction of kvuv doesn't change the existing pass/fail as expected